homeassistant/components/remootio/strings.json

```
@@ -0,0 +1,42 @@
+{
+  "config": {
+    "flow_title": "Remootio",
+    "step": {
+      "user": {
+        "title": "Remootio integration :: Settings",
```

Remootio is already the title of the flow, it doesn't need to be repeated here. Either remove the step title, or shorten it.
```suggestion
Connection settings
```